frontend/src/components/accounts/AccountRow.tsx

```diff
@@ -97,7 +91,9 @@ const AccountRow: React.FC<Props> = React.memo(({ accountId }) => {
                   />
                   <TransactionRowTimer>
                     {t("component.accounts.AccountRow.created_on")}{" "}
-                    {moment(accountInfo.createdAtBlockTimestamp).format("LL")}
+                    {accountInfo.created
+                      ? moment(accountInfo.created.timestamp).format("LL")
+                      : "Genesis"}
```

Comment:
The only thing that was added.
I don't think we need to translate that as there is no chance anybody will scroll down to this account :)